src/core/aws-net-websocket-registry.adb

```
@@ -332,7 +348,10 @@ package body AWS.Net.WebSocket.Registry is
 
       procedure Do_Free (WebSocket : in out Object_Class) is
       begin
-         DB.Free_Or_Defer (WebSocket);
+         DB.Free_Or_Defer (WebSocket.Id);
+      exception
+         when others =>
+            null;
```

likewise, silently dropping exception should not happen.